client/overview/index.js

```diff
@@ -55,10 +56,14 @@ const OverviewPageError = () => {
 const OverviewPage = () => {
 	const {
 		accountStatus,
+		accountStatus: { progressiveOnboarding },
+		// eslint-disable-next-line camelcase
+		accountLoans: { has_active_loan },
```

Comment:
Let's keep the consistency using the camelCase
```suggestion
		accountLoans: { has_active_loan: hasActiveLoan  },
```

Reply:
Silly of me to not think of this! Thanks!

Reply:
Done here!

https://github.com/Automattic/woocommerce-payments/pull/8412/commits/844c9ccb9a6426aedc17151f422aa59941000a9f